pkg/traces/exporter/exporter.go

```diff
@@ -303,6 +332,12 @@ func (t *TraceExporter) Ping() {
 	// to the new token in place.
 	t.ingestAuthToken = string(newToken)
 	t.ingestClientAuthenticator.setToken(t.ingestAuthToken)
+
+	// if the bufspanprocessor doesn't have a child processor
+	// then we don't have an exporter
+	if !t.bufSpanProcessor.HasProcessor() {
+		t.setNewGlobalProvider()
```

Comment:
Why are we doing this?

Reply:
I left a comment to try and explain better. If I understand correctly, we can't create otel batchSpanProcessor unless we have a working client and we have to wait until we get an auth token to get a working client

Reply:
I don't think that was the case before -- I was able to set up a processor with a client with a bad auth token, and it would start working once the good auth token came in -- but maybe something has changed in this update.

Either way I think we can't call `t.setNewGlobalProvider()` here without a check against `t.enabled` first.

Reply:
your correct, my mistake, it was not having the url that was causing it to fail to create exporter, not a bad / missing auth token